packages/fast-tooling-react/src/css-property-editor/property-editor-row.props.ts

```diff
@@ -0,0 +1,62 @@
+import { ManagedClasses } from "@microsoft/fast-jss-manager-react";
+
+export interface CSSPropertyEditorRowClassNameContract {
+    cssPropertyEditorRow?: string;
+    cssPropertyEditorRow_input?: string;
+    cssPropertyEditorRow_inputKey?: string;
+    cssPropertyEditorRow_inputValue?: string;
+}
+
+export interface CSSPropertyEditorRowUnhandledProps
+    extends React.HTMLAttributes<HTMLDivElement> {}
+
+export interface CSSPropertyEditorRowHandledProps
+    extends ManagedClasses<CSSPropertyEditorRowClassNameContract> {
+    /**
+     * The key associated with the value being edited
+     */
+    rowIndex: number;
```

Comment:
Since this isn't necessarily going to be an index but should instead be a unique ID (no matter how this is generated), can we just call this `id`?

Reply:
Actually the comment is what was incorrect, rowIndex is the actual index of the row in the list.  Corrected to "The index of the row".